webview/src/experiments/components/table/styles.module.scss

```diff
@@ -101,7 +101,7 @@ $bullet-size: calc(var(--design-unit) * 4px);
   background: $checkbox-background;
   border-radius: $bullet-size;
   border: calc(var(--border-width) * 1px) solid $checkbox-border;
-  color: $fg-color;
+  color: $row-action-icon-color;
```

Comment:
Q: what does this change?

Reply:
The color of the bullets for unselected experiments. Makes them match unselected experiments in the experiments tree.

Reply:
okay, I see. Could you do the recording across all themes please?

To me it felt okay for them to "blend" with checkboxes in the table tbh.

Reply:
btw, does it apply only to the internal circle?

Reply:
> btw, does it apply only to the internal circle?

yes

Reply:
https://user-images.githubusercontent.com/37993418/196088476-b195342e-c3f7-48b4-8a8e-59fcb5a8829f.mov